include/revng/Pipeline/RegisterAnalysis.h

```diff
@@ -12,25 +12,25 @@ namespace pipeline {
 
 /// Instantiate a global object of this class for each analysis you wish to
 /// register
-template<typename AnalsysisType>
+template<typename AnalysisType>
```

Comment:
Can you push all the fix typo messages to the beginning of the branch?

Reply:
Sure.